### ISSUE TITLE
 "Proposal: Add Decentralized Domain Naming Systems (dDNS) Use-Case Page."

### ISSUE THREAD

**Issue by Spoorthy1423** [ethereum/ethereum-org-website#13489]
### Is your feature request related to a problem? Please describe.

 "Proposal: Add Decentralized Domain Naming Systems (dDNS) Use-Case Page."

### Describe the solution you'd like

# Proposal for Adding a Decentralized Domain Naming Systems (dDNS) Use-Case Page on Ethereum.org

## Introduction

The Ethereum.org homepage currently mentions a "permissionless DNS," but there is no dedicated use-case page to explain and promote decentralized domain naming systems (dDNS). This proposal aims to address this gap by creating a comprehensive use-case page that educates users on the importance of dDNS and its benefits.

### Centralization Issues
The traditional Domain Name System (DNS) is a crucial component of the internet, responsible for translating domain names into IP addresses. However, the current DNS infrastructure is highly centralized, with entities like ICANN controlling domain registration and management. This centralization leads to several problems:
- **Censorship**: Governments and other entities can easily block or take down websites.
- **Server Downtimes**: Centralized servers are vulnerable to outages and attacks.
- **Security Vulnerabilities**: Central points of failure increase the risk of DNS hijacking and other attacks.

### Need for Awareness
Despite the importance of DNS, many users are unaware of its centralization issues and the availability of decentralized alternatives. A dedicated use-case page can help bridge this knowledge gap.

## Proposed Solution

We propose creating a dedicated use-case page for decentralized domain naming systems (dDNS) on Ethereum.org. The page will cover the following topics:

1. **Introduction to DNS**
   - How DNS works and its role in the internet infrastructure.
   - Centralization of DNS and its implications.

2. **Issues with Centralized DNS**
   - Problems caused by centralization, such as censorship, server downtimes, and security vulnerabilities.

3. **Introduction to dDNS**
   - Definition and basic principles of decentralized DNS.
   - Comparison with traditional DNS and its advantages.

4. **How Ethereum Supports dDNS**
   - Overview of the Ethereum blockchain and its suitability for dDNS.
   - Detailed explanation of the Ethereum Name Service (ENS) and its benefits.

5. **Applications of dDNS**
   - Easier crypto payments.
   - Censorship-resistant websites.
   - Reduced server downtimes.
   - Self-custodial social domains.

6. **Tools and Services**
   - Description of services like Unstoppable Domains and ENS.
   - Instructions or links to resources on how to use these tools.

## Benefits

- **User Education**: The new use-case page will educate users on the importance and benefits of dDNS.
- **Increased Adoption**: Promoting dDNS will encourage the adoption of decentralized technologies.
- **Censorship Resistance**: Highlighting dDNS will demonstrate how it can protect against censorship and improve internet resilience.

## Collaboration Offer

I am willing to help start off the creation of this page and contribute to developing its content. I believe this is an important use-case that deserves attention and can significantly benefit the Ethereum community.

## Conclusion

In summary, this proposal aims to create a dedicated use-case page for decentralized domain naming systems (dDNS) on Ethereum.org. This page will educate users, promote the adoption of decentralized technologies, and highlight the benefits of dDNS. I encourage the Ethereum.org team to consider and discuss this proposal.

---

Thank you for your consideration.

Best regards,
[Spoorthy1423]

### Describe alternatives you've considered

I considered adding the dDNS content to the "Web2 vs Web3" or "Introduction to dApps" sections, as well as integrating it into existing use-case pages like those for DeFi or decentralized storage. However, these options might dilute the focus or not provide enough detail. A dedicated use-case page ensures a comprehensive and focused explanation, making it easier for users to understand the benefits and applications of dDNS.

### Would you like to work on this issue?

- [X] Yes
- [ ] No

**Comment by wackerow** [comment]
Hey @Spoorthy1423, thanks for the offer here! I'm a bit confused why you opened a new issue when this is a continuation of #6565, but we can start fresh here if it helps. 

Love the approach you've laid out. Assigning you, PR welcome!

Did you have thoughts on exactly where you'd see this living?

**Comment by minimalsm** [comment]
Hey @Spoorthy1423, thanks for laying out your thinking here. Are you still interested in opening a PR for this?

**Comment by Spoorthy1423** [comment]
Yes!! @minimalsm i am interested in raising another pr, can u please list out the things what can be done, that could be helpful!! Thank you.

**Comment by nloureiro** [comment]
Closing during a backlog sweep: this proposal has been marked 'awaiting PR' since 2024 and none arrived. If you (or anyone) still want to write the dDNS use-case page, we'd be glad to see a fresh issue with an outline — the content standards for use-case pages are in the contributing docs.
